src/js/view/controls/rewind-display-icon.ts

```diff
@@ -10,7 +10,7 @@ export default class RewindDisplayIcon {
         const iconDisplay = element.querySelector('.jw-icon');
 
         this.el = element;
-        this.ui = new UI(iconDisplay).on('click tap enter', function(): void {
+        this.ui = new UI(iconDisplay, { directSelect: true }).on('click tap enter', function(): void {
```

Comment:
We have this util but it is not implemented correctly (direct select is passed in as context rather than the constructor. Can you fix this util and use it in all these places (since `directSelect` is basically a requirement for click handling?) You might also consider not requiring this option when a click listener is added since flag could be added with the click listener so that it always works as expected.

```
export function addClickAction(element, clickAction, payload) {
    const ui = new UI(element);

    if (clickAction) {
        ui.on('click tap enter', function (evt) {
            clickAction(payload || evt);
        }, { directSelect: true });
    }
    return ui;
}
```

Reply:
There are times we want to click through and not prevent a default/bubbling, like the controlbar triggering USER_ACTIVE